index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Mobile Version Skeleton</title>
+</head>
+
+<body>
+    <section class="container">
+        <section class="pg-1">
+            <header>
+                <img class="img" src="img/time.png" alt="time">
+
+                <nav>
+                    <ul>
+                        <li><img src="img/cellular.png" alt="cellular"></li>
+                        <li><img src="img/wifi.png" alt="wifi"></li>
+                        <li><img src="img/capacity.png" alt="capacity"></li>
+                    </ul>
+                </nav>
+            </header>
+
+            <div class="logo">
+
+                <h2 class="my-logo">My Logo</h2>
+
+                <img class="ham" src="img/hamburger.png" alt="hamburger">
+
+            </div>
+
+            <div class="description">
+
+                <div class="name">
+                    <h2>I’m Mariane <br> Glad to see <br> you!</h2>
+
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some
+                        of my work and experience! If you like what you see and have a project you need coded, don’t
+                        hestiate to contact me. </p>
+
+                </div>
+                <div class="connect">
+                    <h2>LET'S CONNECT</h2>
+                </div>
```

Comment:
Please ensure that this section is wrapped in a `section` tag according to the requirement and the content is aligned with flexbox.